src/app/_components/PageHeader.tsx

```diff
@@ -39,43 +34,40 @@ function Title({ children }: TitleProps) {
   )
 }
 
-function PageHeaderImage({ image }: Pick<PageHeaderProps, 'image'>) {
-  const { type, ...rest } = image
+const sharedContainerStyle = 'relative aspect-video lg:aspect-auto lg:w-1/2'
+const sharedImageStyle = 'h-full w-full rounded-lg border border-brand-100'
+const imageSizes = buildImageSizeProp({ startSize: '100vw', md: '660px' })
 
-  if (type === 'static') {
-    const staticImage = rest as StaticImageProps
+function PageHeaderImage({ image }: Pick<PageHeaderProps, 'image'>) {
+  const isDynamicImage = 'src' in image
+  const isStaticImage = 'data' in image
 
+  if (isStaticImage) {
```

Comment:
It's a bit confusing that we have a condition here and also in the `DynamicImage` Component that checks `src` in the `image` object, but couldn't figure out why it wouldn't work if we just put return `DynamicImage`... but I guess something for future refactor

Reply:
If it's confusing then yes we might need a refactor.